src/components/_progress-bar.scss

```diff
@@ -13,19 +14,18 @@
     border-radius: map-get(variables.$radius, lg);
   }
 
-  &.success .progress {
-    background-color: colors.$success;
+  @mixin colors($args...) {
+    @each $name, $color in meta.keywords($args) {
```

Comment:
This loop is not done in the mixin, but rather outside: you loop through names and colors, and call a mixin with each name and color